apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/components/ParametersStep/components/Parameter/hooks/useGetCurrentParameterValue.ts

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { BigNumber, Contract } from 'ethers';
+
+import { ContractGroup, getProtocolContract } from '@sovryn/contracts';
+import { getProvider } from '@sovryn/ethers-provider';
+
+import { defaultChainId } from '../../../../../../../../../../config/chains';
+
+import { useLoadContract } from '../../../../../../../../../../hooks/useLoadContract';
+import { getContractDetails } from '../../../ParametersStep.utils';
+
+export const useGetCurrentParameterValue = (
+  parameter: string,
+  contract: string,
+) => {
+  const [parameterValue, setParameterValue] = useState('');
+  const [contractAddress, setContractAddress] = useState('');
+  const { contractName, contractGroup } = useMemo(
+    () => getContractDetails(contract),
+    [contract],
+  );
+
+  const loadedContract = useLoadContract(
+    contractName,
+    contractGroup as ContractGroup,
+  );
+
+  useEffect(() => {
+    const fetchParameter = async () => {
+      if (!loadedContract || !parameter || parameter === 'custom') {
+        return;
+      }
+
+      const loanTokenSettingsLowerAdminAbi = (
+        await getProtocolContract('loanTokenSettingsLowerAdmin')
+      ).abi;
+      const provider = getProvider(defaultChainId);
+      const contractInstance = new Contract(
+        loadedContract.address,
+        loanTokenSettingsLowerAdminAbi,
+        provider,
+      );
+
+      let parameterResult = '';
+
+      if (parameter === 'checkPause') {
+        parameterResult = await contractInstance.checkPause(parameter);
```

Comment:
This seems a bit weird because you basically call `contract.checkPause('checkPause')`. Also, `iUSDT` check is not working (it cannot find `iUSDT` contract).